packages/react-divider/package.json

```
@@ -0,0 +1,67 @@
+{
+  "name": "@fluentui/react-divider",
+  "version": "0.1.0",
+  "description": "React components for building web experiences",
+  "main": "lib-commonjs/index.js",
+  "module": "lib/index.js",
+  "typings": "lib/index.d.ts",
+  "sideEffects": [
+    "lib/version.js"
+  ],
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/microsoft/fluentui"
+  },
+  "license": "MIT",
+  "scripts": {
+    "build": "just-scripts build",
+    "bundle": "just-scripts bundle",
+    "clean": "just-scripts clean",
+    "code-style": "just-scripts code-style",
+    "just": "just-scripts",
+    "lint": "just-scripts lint",
+    "start": "just-scripts dev:storybook",
+    "start-test": "just-scripts jest-watch",
+    "test": "just-scripts test",
+    "update-snapshots": "just-scripts jest -u"
+  },
+  "devDependencies": {
+    "@fluentui/eslint-plugin": "^1.0.0-beta.2",
+    "@fluentui/react-conformance": "^1.0.0",
+    "@fluentui/scripts": "^1.0.0",
+    "@testing-library/react": "^10.4.9",
+    "@testing-library/react-hooks": "^5.0.3",
+    "@types/enzyme": "3.10.3",
+    "@types/enzyme-adapter-react-16": "1.0.3",
+    "@types/react": "16.9.42",
+    "@types/react-dom": "16.9.10",
+    "@types/react-test-renderer": "^16.0.0",
+    "enzyme": "~3.10.0",
+    "enzyme-adapter-react-16": "^1.15.0",
+    "react": "16.8.6",
+    "react-app-polyfill": "~1.0.1",
+    "react-dom": "16.8.6",
+    "react-test-renderer": "^16.3.0"
+  },
+  "dependencies": {
+    "@fluentui/react-hooks": "^8.0.0-beta.14",
+    "@fluentui/react-make-styles": "^0.2.7",
+    "@fluentui/react-theme-provider": "^1.0.0-beta.26",
```

This shouldn't be necessary

```suggestion
    "@fluentui/react-theme-provider": "^1.0.0-beta.26",
```

---

Updated